scripts/cli/index.js

```diff
@@ -211,23 +211,21 @@ async function main() {
   await tps(api, keyring, n_accounts, init_bars[0], init_bars[1], fast); // base currency transfer sanity-check
   await distributePoly(api, keyring, master_keys.concat(signing_keys).concat(claim_keys), transfer_amount, init_bars[2], init_bars[3], fast);
   // Need to wait until POLY has been distributed to pay for the next set of transactions
-  await blockTillPoolEmpty(api, n_accounts);
+  await blockTillPoolEmpty(api);
 
   let issuer_dids = await createIdentities(api, master_keys, "issuer", prepend, init_bars[4], init_bars[5], fast);
   await addSigningKeys(api, master_keys, issuer_dids, signing_keys, init_bars[6], init_bars[7], fast);
   await addSigningKeyRoles(api, master_keys, issuer_dids, signing_keys, init_bars[8], init_bars[9], fast);
   await issueTokenPerDid(api, master_keys, issuer_dids, prepend, init_bars[10], init_bars[11], fast);
   let claim_issuer_dids = await createIdentities(api, claim_keys, "claim_issuer", prepend, init_bars[12], init_bars[13], fast);
   // Need to wait until identites have been created before we use them
-  await blockTillPoolEmpty(api, n_accounts);
```

Comment:
Won't `addClaimIssuersToDids` fail if the claim issuers DIDs haven't been created first?

Reply:
Almost everything will fail if DIDs have not been created first, therefore, I add `await blockTillPoolEmpty` in the `createIdentities` function itself :).